src/main/webapp/script.js

```diff
@@ -55,3 +56,139 @@ window.addEventListener("load", function(){
   function a(a,b){var c=/^(?:file):/,d=new XMLHttpRequest,e=0;d.onreadystatechange=function(){4==d.readyState&&(e=d.status),c.test(location.href)&&d.responseText&&(e=200),4==d.readyState&&200==e&&(a.outerHTML=d.responseText)};try{d.open("GET",b,!0),d.send()}catch(f){}}var b,c=document.getElementsByTagName("*");for(b in c)c[b].hasAttribute&&c[b].hasAttribute("data-include")&&a(c[b],c[b].getAttribute("data-include"));
 });
 
+function displayPotentialMatchInfo(pmID) {
+  fetch('/user-data?id=' + pmID).then(response => response.json()).then((userinfo) => {
+    let name = userinfo.name;
+    let bio = userinfo.bio;
```

Comment:
nit: probably not necessary to store variables, up to you though.

Reply:
I think it'll make things easier in the future when blobstore is being used so I am gonna leave it for now